pkg/xray/provider_test.go

```diff
@@ -82,6 +82,37 @@ func testAccDeleteRepo(t *testing.T, repo string) {
 	}
 }
 
+// Create a project. It will be used in the tests
+func testAccCreateProject(t *testing.T, projectKey string, projectName string) {
+	restyClient := getTestResty(t)
+
+	type Project struct {
+		DisplayName string `json:"display_name"`
+		Description string `json:"description"`
+		ProjectKey  string `json:"project_key"`
+	}
+
+	project := Project{}
+	project.DisplayName = projectName
+	project.Description = "Project created by TF provider test"
+	project.ProjectKey = projectKey
+	response, errProject := restyClient.R().SetBody(project).Post("/access/api/v1/projects")
```

Comment:
You're not checking the actual error here

Reply:
Yes, this is correct. I only care if the project wasn't created at this point, and if it wasn't, stop the test and printout the error.

Reply:
`errProject` would also mean the project wasn't created. You're skipping a whole class of problems. This has to be checked.

Reply:
This comment is my only required change. Save yourself trouble and don't bother debating:
```go
if errProject != nil {
  return errProject
}
```